spec/requests/providers/means/identify_types_of_outgoings_controller_spec.rb

```diff
@@ -53,20 +53,24 @@
       }
     end
 
-    it "does not add transaction types to the application" do
-      expect { request }.not_to change(LegalAidApplicationTransactionType, :count)
-    end
+    context "when no transaction types selected" do
+      let(:transaction_type_ids) { [] }
 
-    it "displays an error" do
-      request
-      expect(response.body).to match("govuk-error-summary")
-      expect(unescaped_response_body).to match(I18n.t("providers.means.identify_types_of_outgoings.update.none_selected"))
-      expect(unescaped_response_body).not_to include("translation missing")
-    end
+      it "does not add transaction types to the application" do
+        expect { request }.not_to change(LegalAidApplicationTransactionType, :count)
+      end
 
-    it "returns http success" do
-      request
-      expect(response).to have_http_status(:ok)
+      it "displays an error" do
+        request
+        expect(response.body).to match("govuk-error-summary")
+        expect(unescaped_response_body).to match(I18n.t("providers.means.identify_types_of_outgoings.update.none_selected"))
+        expect(unescaped_response_body).not_to include("translation missing")
```

Comment:
If we have `I18n` configured to raise errors on missing translations, do we need this expectation?

Reply:
Not something I changed, just moved i think, but happy to discuss. development will raise when you hit such a page but use of interpolation in some views and named scopes means I18n-missing gem won't pick up all missing translations